src/main/java/org/veupathdb/service/eda/ss/model/db/FilteredResultFactory.java

```diff
@@ -127,6 +133,38 @@ public static void produceTabularSubset(DataSource dataSource, String appDbSchem
     }, FETCH_SIZE_FOR_TABULAR_QUERIES);
   }
 
+  /**
+   * Writes to the passed output stream a "tabular" result.  Exact format depends on the passed
+   * responseType (JSON string[][] vs true tabular). Each row is a record containing
+   * the primary key columns and requested variables of the specified entity.
+   *
+   * @param study           study context
+   * @param outputEntity    entity type to return
+   * @param outputVariables variables requested
+   * @param filters         filters to apply to create a subset of records
+   */
+  public static void produceTabularSubsetFromFile(Study study, Entity outputEntity,
```

Comment:
maybe the method name should be `...FromFiles`.  as it is, i am looking for an input file